crates/editor/src/element.rs

```diff
@@ -4153,7 +4153,20 @@ fn render_inline_blame_entry(
     let relative_timestamp = blame_entry_relative_timestamp(&blame_entry);
 
     let author = blame_entry.author.as_deref().unwrap_or_default();
-    let text = format!("{}, {}", author, relative_timestamp);
+    let summary_enabled = ProjectSettings::get_global(cx)
+        .git
+        .show_inline_commit_summary();
+
+    let text = if summary_enabled && blame_entry.summary.is_some() {
+        format!(
+            "{}, {} - {}",
+            author,
+            relative_timestamp,
+            blame_entry.summary.as_ref().unwrap()
+        )
+    } else {
+        format!("{}, {}", author, relative_timestamp)
+    };
```

Comment:
Slightly nitpicky, but I would prefer to avoid the `.unwrap`, since that always kicks off the alarms in my head.

```suggestion
    let text = match blame_entry.summary.as_ref() {
        Some(summary) if summary_enabled => {
            format!("{}, {} - {}", author, relative_timestamp, summary)
        }
        _ => format!("{}, {}", author, relative_timestamp),
    };
```

Reply:
Not nit picky, I felt dirty about that 😂